rest_spec.md

```diff
@@ -63,15 +63,15 @@ These APIs allow the admin user to create new users with access to certain resou
 
 | HTTP Verb | URL | Request Body | Response Code | Response Body | Function |
 | --- | --- | :---  | :---  | :---  | :--- |
-| POST | `/users` | `{"roleSetId": "client\|admin","resourceId": "*\|some id"}` | See Collection responses | `{“bearerToken”: "token"}` | Create a new user and assign a bearer token, setup internal management of the user-id that does not require saving the bearer token and attached the named `roleSet` for the `resource-id` to the new user |
-| DELETE | `/users/user-id` | `{"roleSetId": "client\|admin", "resourceId": "*\|<some-user-id>"}` | See Item responses | `{"bearerToken": "token"}` | Removes the user and all access rules from the system |
+| POST | `/users/user-id/permissions` | `{"userId": "user-id", "roleSetId": "client\|admin","resourceId": "*\|some id"}` | See Collection responses | `{"userId": "user_id", “bearerToken”: "token"}` | Create a new user and assign a bearer token and user-id, setup internal management of the user-id that does not require saving the bearer token and attached the named `roleSet` for the `resource-id` to the new user |
```

Comment:
I agree. `POST /users` can set some initial state of the new user's permission and return the user-id. Then creating or deleting permissions will be sufficient but we will also need `DELETE /users/<user-id>` I can make the changes without a PR and you can review.